solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrJdkClient.java

```diff
@@ -64,6 +62,12 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * A SolrClient implementation that communicates to a Solr server using the built-in Java 11+ Http
+ * Client. This client is targeted for those users who wish to minimize application dependencies.
+ * This client will connect to solr using Http/2 but can seamlessly downgrade to Http/1.1 when
+ * connecting to Solr hosts running on older versions.
+ */
 public class HttpSolrJdkClient extends HttpSolrClientBase {
```

Comment:
If I suggested the "Jdk" between Solr & Client then I misspoke; I recall HttpJdkSolrClient or "JdkHttpSolrClient" (I like the second) because "SolrClient" is the important base type here.

Reply:
Cool; that was my second preference.